kloppy/domain/models/pitch.py

```diff
@@ -32,34 +39,14 @@ class PitchDimensions:
 
     x_dim: Dimension
     y_dim: Dimension
-    x_per_meter: float = None
-    y_per_meter: float = None
+    length: float = None
+    width: float = None
 
-    @classmethod
```

Comment:
@koenvo I deleted this default method, and also set the serializer for XML to have None as pitch dimensions. 

I think if there are no coordinate system or pitch dimensions is better to not assume anything / not have any default. What do you think?